### ISSUE TITLE
Error: Could not complete SSL handshake. 1

### ISSUE THREAD

**Issue by ghost** [tbartelmess/NRPE-OSX#2]
I could not get our nagios server (version 4.0.8) to talk to the nrpe daemon, always getting this error on the server side:

```
CHECK_NRPE: Received 0 bytes from daemon.  Check the remote server logs for error messages.
```

... and the nrpe log on the Mac would just state:

```
9.4.2015 13.14.32,266 nrpe[3227]: Error: Could not complete SSL handshake. 1
```

Disabling SSL in nrpe solved the problem (com.nagios.nrpe.plist):

``` xml
<string>-n</string>
```

**Comment by tbartelmess** [comment]
What OS are you running on?

**Comment by ghost** [comment]
This was on 10.8.5

**Comment by tbartelmess** [comment]
And the server is configured to use SSL?

**Comment by ghost** [comment]
Good point. This is the nagios4 package from FreeBSD (10.1) and now that you mention it, I don't think it even supports SSL. I haven't found a definitive answer, but pkg info on nagios4 says:

```
Shared Libs required:
    libpng16.so.16
    libltdl.so.7
    libjpeg.so.8
    libgd.so.5
```

... so I'm guessing this issue has nothing to do with NRPE-OSX :)
